src/lightning/fabric/utilities/cloud_io.py

```diff
@@ -51,6 +56,9 @@ def _load(
             weights_only=weights_only,
         )
     if str(path_or_url).startswith("http"):
+        if weights_only is None:
```

Comment:
```suggestion
        if weights_only is None and _TORCH_GREATER_EQUAL_2_6:
```

Reply:
@Borda 

[`torch.load_state_dict_from_url`](https://docs.pytorch.org/docs/2.8/hub.html#torch.hub.load_state_dict_from_url) defaults to `weights_only=False` even for 2.8. Should we still default to `weights_only=True` in this case?

For cases of `torch.load`, I am not modifying `weights_only` if the function receives `weights_only=None` as the underlying `torch.load` function will handle defaults. We cannot do that with `torch.load_state_dict_from_url` as the argument is `weights_only: bool = False`.

Reply:
ok, lest set our default only if there is none yet